src/main/scala/code/snippet/UserSnips.scala

```diff
@@ -147,6 +150,103 @@ object ProfileLocUser extends UserSnippet {
   }
 }
 
+object UserConfirmation extends Loggable {
+  def render = {
+    val message = (for {
```

Comment:
Ok si pones el tipo y heredas de SNippetHelper como mencion arriba, el metodo de abajo puede convertirse a

``
for {
  user <- Site.emailConfirmation.currentValue ?~ "Ocurrio un error. Porfavor intente nuevamente."
} yield {
  user.verified(true),save(true)
  "data-name=message" #> "Felicidades su cuenta ha sido creada con exito. Ahora usted puede ingresar al sistema."
}